OtripleS.Web.Api.Tests.Unit/Services/Foundations/Attendances/AttendanceServiceTests.cs

```diff
@@ -88,9 +88,10 @@ private static Filler<Attendance> GetAttendanceFiller(DateTimeOffset dateTime)
             var attendance = new Filler<Attendance>();
 
             attendance.Setup()
+                .OnProperty(attendance => attendance.Status).Use(AttendanceStatus.Present)
                 .OnType<DateTimeOffset>().Use(dateTime);
 
             return attendance;
-        }
+        }        
```

Comment:
Run code clean up on your solution.

Reply:
Sincere thanks for your time & advice.  I failed to mark this earlier as I had done the commit from here browser window. I didn't have any experience of marge conflict and my GitHub Desktop & Visual Studio were not showing the conflict matter! Thanks again for your video titled [Performing Code Reviews in Github Codespaces](https://youtu.be/50BXxp9-cDY)!